fe/fe-core/src/test/java/com/starrocks/sql/plan/PlanFragmentTest.java

```diff
@@ -5517,4 +5517,11 @@ public void testSemiJoinReorder() throws Exception {
                 "  |  equal join conjunct: 5: v2 = 11: v1\n" +
                 "  |  other predicates: (10: expr) OR (11: v1 IS NOT NULL)"));
     }
+
+    @Test
+    public void testMd5sum() throws Exception {
```

Comment:
md5sum constant folding result is wrong. But it feels that this UT is not covering this problem?